examples/chartmuseum/componentplan.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: ComponentPlan
+metadata:
+  name: chartmuseum
+  namespace: kubebb-system
+spec:
+  approved: true
+  name: chartmuseum
+  version: 3.10.1
+  component:
+    name: kubebb.chartmuseum
```

Comment:
The format of the component name is `<repository-name>.<chart-name>`.

So, if you use `repository_chartmuseum.yaml` to create `Repository`, the component name should be `chartmuseum.chartmuseum`.

Reply:
I deployed this component from kubebb repository first,then I used this component to create repository.Why is this component called chartmuseum.chartmuseum instead of kubebb.chartmuseum.

Reply:
Er..